docs/config/authentication.md

```diff
@@ -53,6 +53,7 @@ auth: {
       passFieldSelector: '<CSS selector of password input field>',
       logonButtonSelector: '<CSS selector of submit button>',
       idpSelector: '<CSS selector of login link>',
+      conditional: '<boolean indicating whether to expect conditional steps>',
```

Comment:
lets use the already existing practice and have conditionalSelector for the conditional button.